README.md

```diff
@@ -0,0 +1,44 @@
+# unflatten-recursive [![CircleCI](https://circleci.com/gh/UrbanCompass/unflatten-recursive.svg?style=shield)](https://circleci.com/gh/UrbanCompass/unflatten-recursive)
+
+Unflattens object recursively while preserving arrays. Contents of arrays are unflattened.
+
+Note: UNLIKE OTHER OPEN SOURCE LIB, this lib also looping through arrays and unflatten the array
```

Comment:
Also we should always specify WHICH "other open source lib", not just say "UNLIKE OTHER"